Gems/Maestro/Code/Source/Cinematics/AnimSerializer.cpp

```diff
@@ -16,9 +16,32 @@ void AnimSerializer::ReflectAnimTypes(AZ::ReflectContext* context)
 {
     if (auto serializeContext = azrtti_cast<AZ::SerializeContext*>(context); serializeContext != nullptr)
     {
+        auto CAnimParamTypeVersionConverter = [](AZ::SerializeContext& context,
+            AZ::SerializeContext::DataElementNode& rootElement) -> bool
+        {
+            if (rootElement.GetVersion() < 2)
+            {
+                constexpr AZ::u32 NameCrc = AZ_CRC_CE("Name");
+                AZStd::string oldName;
+                if (!rootElement.GetChildData(NameCrc, oldName))
+                {
+                    AZ_Error("AnimSerializer", false, R"(Unable to read "Name" field with type AZStd::string from instance of CAnimParamType)"
+                        " with version number %u", rootElement.GetVersion());
+                    return false;
+                }
+
+                decltype(CAnimParamType::m_name) newNameType(oldName);
+                // Remove the "Name" field with AZStd::string type and re-add it with
+                // the AZStd::basic_string<char, AZStd::char_traits<char>, AZStd::stateless_allocator> type
+                rootElement.RemoveElementByName(NameCrc);
+                rootElement.AddElementWithData(context, "Name", newNameType);
+            }
+
+            return true;
+        };
         // Reflection for Maestro's AZ_TYPE_INFO'ed classes
         serializeContext->Class<CAnimParamType>()
-            ->Version(1)
+            ->Version(2, CAnimParamTypeVersionConverter)
```

Comment:
Is this change related to this PR?

Reply:
No, this is related to a TrackView issue.

Reply:
Consider moving it to another PR.